.travis.yml

```diff
@@ -3,6 +3,8 @@ node_js:
   - 5
 os:
   - linux
+script:
+  npm test && python node_modules/travis-weigh-in/weigh_in.py dist/onfido.min.js
```

Comment:
I changed this.

Interesting that we can run Python in the node travis environment - we could use my Python surge/travis script instead of bash.